tests/kafkatest/benchmarks/streams/streams_simple_benchmark_test.py

```diff
@@ -50,27 +54,38 @@ def test_simple_benchmark(self, test, scale):
         self.zk = ZookeeperService(self.test_context, num_nodes=1)
         self.zk.start()
         self.kafka = KafkaService(self.test_context, num_nodes=scale, zk=self.zk, version=DEV_BRANCH, topics={
-            'simpleBenchmarkSourceTopic' : { 'partitions': scale, 'replication-factor': self.replication },
-            'countTopic' : { 'partitions': scale, 'replication-factor': self.replication },
-            'simpleBenchmarkSinkTopic' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic1KStreamKStream' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic2KStreamKStream' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic1KStreamKTable' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic2KStreamKTable' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic1KTableKTable' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic2KTableKTable' : { 'partitions': scale, 'replication-factor': self.replication },
-            'yahooCampaigns' : { 'partitions': 20, 'replication-factor': self.replication },
-            'yahooEvents' : { 'partitions': 20, 'replication-factor': self.replication }
+            'simpleBenchmarkSourceTopic1' : { 'partitions': scale, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" },
+            'simpleBenchmarkSourceTopic2' : { 'partitions': scale, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" },
+            'simpleBenchmarkSinkTopic' : { 'partitions': scale, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" },
+            'yahooCampaigns' : { 'partitions': 20, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" },
+            'yahooEvents' : { 'partitions': 20, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" }
         })
         self.kafka.log_level = "INFO"
         self.kafka.start()
- 
+
+        self.key_skew = 0
```

Comment:
Just for my own information - does `key_skew` adjust keys to have a configurable un-even distribution?